federatedscope/main.py

```diff
@@ -10,37 +10,36 @@
 from federatedscope.core.auxiliaries.data_builder import get_data
 from federatedscope.core.auxiliaries.utils import setup_seed, update_logger
 from federatedscope.core.auxiliaries.worker_builder import get_client_cls, get_server_cls
-from federatedscope.core.configs.config import global_cfg
+from federatedscope.core.configs.config import global_cfg, CN
 from federatedscope.core.fed_runner import FedRunner
-from yacs.config import CfgNode
+from federatedscope.nlp.configs.cfg_client import extend_cfg_client
 
 if os.environ.get('https_proxy'):
     del os.environ['https_proxy']
 if os.environ.get('http_proxy'):
     del os.environ['http_proxy']
 
+os.environ['TOKENIZERS_PARALLELISM'] = 'false'
+
+
 if __name__ == '__main__':
     init_cfg = global_cfg.clone()
     args = parse_args()
     init_cfg.merge_from_file(args.cfg_file)
     init_cfg.merge_from_list(args.opts)
-
     update_logger(init_cfg)
     setup_seed(init_cfg.seed)
 
-    # federated dataset might change the number of clients
-    # thus, we allow the creation procedure of dataset to modify the global cfg object
     data, modified_cfg = get_data(config=init_cfg.clone())
     init_cfg.merge_from_other_cfg(modified_cfg)
-
     init_cfg.freeze()
 
     # allow different settings for different clients
-    # cfg_client.merge_from_file(args.cfg_client)
     if args.cfg_client is None:
         cfg_client = None
     else:
-        cfg_client = CfgNode.load_cfg(open(args.cfg_client, 'r'))
+        cfg_client = CN.load_cfg(open(args.cfg_client, 'r'))
```

Comment:
The dataset Graph-DT also uses `args.cfg_client`, which may cause certain error here. 
please see the comments in `federatedscope/nlp/configs/cfg_client.py`